src/audio/pipeline/pipeline-stream.c

```diff
@@ -52,6 +52,10 @@ pipeline_should_report_enodata_on_trigger(struct comp_dev *rsrc,
 	if (pipe_source && dev_comp_type(pipe_source) == SOF_COMP_DAI)
 		return false;
 
+	/* source pipeline may not be active since priority is not higher than current one */
```

Comment:
@RanderWang what issue are you seeing because of this in Linux? AFAICT we do not use the pipeline priority to start a pipeline today

Reply:
@ranj063  Sure, currently host driver doesn't set priority but it should set it in future.

Reply:
please check it. This function is only for echo reference capture.